src/components/Footer/LogoFooter.tsx

```diff
@@ -0,0 +1,16 @@
+import Image from "next/image";
+import LogoFooterMobile from "../../../public/images/LogoFooterMobile.png";
+
+const LogoFooter: React.FC = () => {
+  return (
+    <div className="mb-[20px] mt-[20px] flex w-full items-center justify-center">
+      <Image
+        src={LogoFooterMobile}
+        alt="Logo"
+        className="h-auto w-[83px]"
+        priority
```

Comment:
Since priority is being used, ensure that this image is crucial for above-the-fold content. Otherwise, consider removing it to improve loading performance.